satpy/etc/readers/fci_l2_nc.yaml

```diff
@@ -1310,7 +1421,9 @@ datasets:
     resolution: 32000
     file_type: nc_fci_asr
     file_key: reflectance_mean
-    long_name: mean_reflectance_in_segment
+    long_name: TOA Bidirectional Reflectance segment mean
+    standard_name: toa_bidirectional_reflectance
```

Comment:
also add `units: %`, set to none in netcdf file.

Reply:
done